cocos/renderer/gfx-metal/MTLDevice.mm

```diff
@@ -49,8 +49,8 @@ of this software and associated documentation files (the "Software"), to deal
 
 CCMTLDevice::CCMTLDevice() {
     _clipSpaceMinZ = 0.0f;
-    _screenSpaceSignY = 1.0f;
-    _UVSpaceSignY = 1.0f;
+    _screenSpaceSignY = -1.0f;
+    _clipSpaceSignY = 1.0f;
```

Comment:
need to update this for vk device too

Reply:
> need to update this for vk device too

Yes, but I didn’t bring a windows computer today, so I’m not sure about the changes.

Reply:
for vulkan, both clip and screen space sign Y should be -1

Reply:
> for vulkan, both clip and screen space sign Y should be -1

I remember that I also changed other places, so the vulkan code has not been submitted yet

Reply:
> Yes, but I didn’t bring a windows computer today, so I’m not sure about the changes.

You can use Android device to test.